### PR TITLE
examples/synthesis_json.py can take multiple file path inputs.

### PR DESCRIPTION
**Pull request by juanchopanza**
Runs analysis for each file path separately.

TODO: Write out JSON files for each iteration
TODO: Find naming scheme for when files do not have m-type in their name
TODO: Store results of analyses in list to user can use them in
interactive session

### CODE REVIEW

**Comment by juanchopanza** [comment]
@lidakanari This might be of interest.

**Comment by lidakanari** [comment]
TODO: Find naming scheme for when files do not have m-type in their name

-> That would be my comment as well, in order to make it work for biological data. Could we make the example take as input a datapath (or multiple) and the corresponding type for each directory if it is provided?

**Comment by juanchopanza** [comment]
Yes, I am trying to figure out the least confusing way of providing this functionality via the commandline. I hope to have something ready later today.

File: examples/synthesis_json.py [hunk 1]
```diff
@@ -113,31 +114,33 @@ def transform_package(mtype, files, components, feature_list):
...
-    d_path = args.datapath
+    data_dirs = args.datapaths
 
     mtype_files = defaultdict(list)
 
-    for f in get_morph_files(d_path):
-        mtype_files[get_mtype(f)].append(f)
-
     flist = [["soma_radius", "radius", "soma", None, None, None],
              ["n_neurites", "number", "basal_dendrite", 1, None,
```
Comment (liesbethvanherpe) [inline examples/synthesis_json.py]: For my education: what do "n_neurites" and "number" mean here?

(n_neurites is probably number of neurites, but what is number?)

Comment (lidakanari) [inline examples/synthesis_json.py]: The first is the NeuroM name, the second is the morphine readable name chosen in this commit:

https://bbpcode.epfl.ch/code/#/c/17824/

They both mean the same thing, but we need different names to be recognizable by the corresponding software.

**Pull request merged**